### PR TITLE
Modify tc and mm_heapinfo.c based on svace

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

File: apps/examples/testcase/le_tc/kernel/tc_pthread.c [hunk 1]
```diff
@@ -895,7 +895,7 @@ static void tc_pthread_pthread_sem_take_give(void)
 	TC_ASSERT_EQ("pthread_sem_take", ret_chk, OK);
 
 	sem_getvalue(&sem, &get_value);
-	TC_ASSERT_EQ("sem_getvalue", get_value, VAL_TWO);
+	TC_ASSERT_EQ_CLEANUP("sem_getvalue", get_value, VAL_TWO, pthread_sem_give(&sem));
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/kernel/tc_pthread.c]: not destory?

File: apps/examples/testcase/le_tc/kernel/tc_pthread.c [hunk 2]
```diff
@@ -1281,7 +1281,7 @@ static void tc_pthread_pthread_cond_signal_wait(void)
 	ret_chk = pthread_create(&pthread_waiter, NULL, thread_cond_signal, NULL);
 	TC_ASSERT_EQ("pthread_create", ret_chk, OK);
 
-	TC_ASSERT_EQ("pthread_mutex_lock", g_cond_sig_val, VAL_ONE);
+	TC_ASSERT_EQ_CLEANUP("pthread_mutex_lock", g_cond_sig_val, VAL_ONE, pthread_mutex_unlock(&g_mutex));
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/kernel/tc_pthread.c]: not destory?

File: os/mm/mm_heap/mm_heapinfo.c [hunk 3]
```diff
@@ -274,6 +274,7 @@ void heapinfo_exclude_stacksize(void *stack_ptr)
 	node = (struct mm_allocnode_s *)(stack_ptr - SIZEOF_MM_ALLOCNODE);
 	rtcb = sched_gettcb(node->pid);
 
+	DEBUGASSERT(rtcb);
```
Comment (sunghan-chang) [inline os/mm/mm_heap/mm_heapinfo.c]: DEBUG is enough?

Comment (jeongchanKim) [reply os/mm/mm_heap/mm_heapinfo.c]: I think it is enough

**Pull request merged**